### PR TITLE
Update noble.d.ts

### PR DESCRIPTION
**Pull request by hansmbakker**
Fix type for existing services property

### CODE REVIEW

**Comment by swook** [comment]
:+1: thanks

**Comment by wind-rider** [comment]
Thanks for reviewing my PR, will it be automatically merged now?

Or did you need to put a :+1: reaction at the @dt-bot message itself? (there is an emoticon that you can click on in the header "dt-bot commented 20 hours ago" at the right-hand side)

**Comment by swook** [comment]
I've done as you said, and I hope this is enough! Please do bump if this isn't auto-merged.

**Comment by wind-rider** [comment]
oh, I just read that apparently somebody from the DefinitelyTyped team will do it :)

http://definitelytyped.org/guides/pull-request.html

**Comment by wind-rider** [comment]
@swook please have another look, I added another commit for the Buffer type

File: noble/noble.d.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // Type definitions for noble
 // Project: https://github.com/sandeepmistry/noble
 // Definitions by: Seon-Wook Park <https://github.com/swook>
+// Additions by: Hans Bakker <https://github.com/wind-rider>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```
Comment (wind-rider) [inline noble/noble.d.ts]: also - when I install the typings for node they are in /// \<reference path="../node/**index**.d.ts" />. Could you have a look at that?

Comment (swook) [inline noble/noble.d.ts]: I cannot find any other definition file which refers to an `../node/index.d.ts` and as far as I know the node definitions should be in `../node/` with version number sufficies.

I would suggest reverting to `../node/node.d.ts` as this works for both the CI tool as well as a clean setup.

**Comment by swook** [comment]
Thank you for the `Buffer` class update, after changing the node definition reference, this PR is good to go.

**Comment by wind-rider** [comment]
@swook done

**Comment by vvakame** [comment]
please ping me again when I can merge this PR.

**Comment by swook** [comment]
@vvakame lgtm

**Pull request merged**